src/main/groovy/edu/oregonstate/mist/catalogapitest/CatalogAPITestApplication.groovy

```diff
@@ -0,0 +1,37 @@
+package edu.oregonstate.mist.catalogapitest
+
+import com.fasterxml.jackson.databind.SerializationFeature
+
```

Comment:
remove spaces between imports.

this is a suggestion, but i prefer to order imports starting with project dependencies (e.g. edu.oregonstate.*), then dropwizard dependencies (io.dropwizard.*), and then miscellaneous others.